test/system/autocomplete_component_test.rb

```diff
@@ -83,6 +111,5 @@ def test_event_handler
     accept_alert "Selected vinyl" do
       find(".Polaris-OptionList-Option", text: "Vinyl").click
     end
-    assert_selector ".Polaris-OptionList-Option--select", text: "Vinyl"
```

Comment:
Instead of removing we could still check the value of the field:
```rb
assert_field "tags", with: "Vinyl"
```

Reply:
You're right 😄 👍